pkg/solana/config/config.go

```diff
@@ -166,8 +166,10 @@ func (n *Node) ValidateConfig() (err error) {
 	}
 	if n.URL == nil {
 		err = errors.Join(err, config.ErrMissing{Name: "URL", Msg: "required for all nodes"})
+	} else if n.URL.String() == "" {
+		err = errors.Join(err, config.ErrMissing{Name: "URL", Msg: "required for all nodes"})
```

Comment:
As Amit suggested, should we use `config.ErrEmpty` instead of `config.ErrMissing` ?